python/GafferSceneUI/LightEditor.py

```diff
@@ -368,9 +369,6 @@ def __buttonPress( self, pathListing, event ) :
 			if cellColumn == columns[i] :
 				columnIndex = i
 
-		if columnIndex <= 0 :
-			return False
-
 		cellPath = pathListing.pathAt( event.line.p0 )
```

Comment:
This is a pre-existing problem, but we're still showing the menu even when there's no path (when you right click in empty space). I think it'd be better to either show no menu, or to grey out the items?

Reply:
Good point, and it is also clearing any existing selection when clicking on an empty cell which I don't think is a good thing.

Reply:
Fixed in 2c05467872d4612bb15926e93eb3f4c4c414b794